OSDOCS-2443: Adding support for installing a cluster to AWS China regions (#35550)
openshift/openshift-docs: modules/cluster-entitlements.adoc

[IMPORTANT]
====

> If your cluster cannot have direct internet access, you can perform a restricted network installation on some types of infrastructure that you provision. During that process, you download the required content and use it to populate a mirror registry with the packages that you need to install a cluster and generate the installation program. With some installation types, the environment that you install your cluster in will not require internet access. Before you update the cluster, you update the content of the mirror registry.

Reviewer: ```suggestion
If your cluster does not have direct internet access, you can perform a restricted network installation. During that process, you download the required content and use it to populate a mirror registry with the packages that you need to install a cluster and generate the installation program. With some installation types, the environment that you install your cluster in will not require internet access. Before you update the cluster, you update the content of the mirror registry.
```

Attempted to simplify this content. It's very hard to read, so it maybe an issue for translation. I attempted to simplify without losing much of the meaning behind each sentence, but if you could also simplify the second sentence that'd be great!

Author: I removed `that you need to install a cluster and generate the installation program` and shortened it to `installation packages` as this does not change the technically accuracy of the sentence.

I do agree that the first sentence is awkward, but I do not have enough background on restricted installations to be sure that the content would remain technically accurate if we remove `on some types of infrastructure that you provision`. This is content that was previously approved and is reused across many (if not all) of the installation assemblies.

Further, I suspect that `cannot` is accurate. I am speculating that the author chose `cannot` because because an organizational policy dictates that the cluster cannot have internet access.
